### PR TITLE
Cleanup of social media profiles (twitter / facebook)

### PR DESCRIPTION
**Pull request by georgengelmann**

### CODE REVIEW

**Comment by kenman345** [comment]
Why are we removing these? Are they dead?

File: _data/directories.yml [hunk 1]
```diff
@@ -26,8 +26,6 @@ websites:
 - name: ELIPAY
   url: https://elipay.com/
   img: elipay.png
-  twitter: eligmacom
```
Comment (kenman345) [inline _data/directories.yml]: the twitter account for this still exists. The facebook though does not. Please put the twitter account back.

File: _data/payments.yml [hunk 2]
```diff
@@ -266,8 +266,6 @@ websites:
 - name: ELIPAY
   url: https://elipay.com/
   img: elipay.png
-  twitter: eligmacom
```
Comment (kenman345) [inline _data/payments.yml]: Put this twitter account back too

**Comment by georgengelmann** [comment]
These are broken links. EliPay handle is back to normal, but only because I asked them about it on Telegram

**Pull request merged**